api_v3/lib/syndication/KalturaSyndicationFeedRenderer.php

```diff
@@ -408,6 +408,18 @@ public function getNextEntryByPlaylistType()
 		}
 	}
 
+	protected function shouldUseDynamicPlaylist()
+	{
+		$syndicationMap = kConf::getMap('syndication');
```

Comment:
1. Better practice - 
kConf::get('playlistExecutionLimitPartners','syndication',array());
And then just check if the partner is in this array.

2. Add the map name + section name  as const to syndycation class

Reply:
done